docs/Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
-ruby RUBY_VERSION
+ruby '2.7.1'
```

Comment:
yes, according to the bundler docs, you can specify ruby version using this syntax making it easier for people that don't want to create another rvm env for this project if they have another otherwise compatible version. 
https://bundler.io/gemfile_ruby.html

the only way, perhaps, this could backfire is if there is a gem with a strict dependency, in which case, specifying the engine/interpreter may be a way to make the gem compatible with an incompatible ruby version. I doubt that's a problem, but thought I'd brain dump on that anyway.

Reply:
Thanks for this @DavidCPorter. 👍

Reply:
@rubenlorenzo fyi

Reply:
So, it seems like having 2 different versions of ruby like 2.7.0 and 2.7.1 won't be an issue then? @leecalcote and @DavidCPorter , right?
Or shall we change the versions in other repos as well, to 2.7.1?

Reply:
Consistently would be really nice. Perhaps, adjusting this repo to 2.7.0 is easiest.